апр. 22, 2022 9:45:11 AM net.sf.l2j.Config initProperties
SEVERE: An error occured loading './config/aelia.properties' config.
java.io.FileNotFoundException: .\config\aelia.properties (Не удается найти указанный файл)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at net.sf.l2j.commons.config.ExProperties.load(ExProperties.java:28)
	at net.sf.l2j.Config.initProperties(Config.java:735)
	at net.sf.l2j.Config.loadAelia(Config.java:53)
	at net.sf.l2j.Config.loadGameServer(Config.java:1420)
	at net.sf.l2j.gameserver.GameServer.<init>(GameServer.java:144)
	at net.sf.l2j.gameserver.GameServer.main(GameServer.java:118)

апр. 22, 2022 9:50:25 AM net.sf.l2j.commons.concurrent.ThreadPool$TaskWrapper run
SEVERE: Exception in a ThreadPool task execution.
java.lang.NullPointerException
	at net.sf.l2j.gameserver.data.manager.ZoneManager.getAllZones(ZoneManager.java:303)
	at net.sf.l2j.gameserver.data.manager.ZoneManager.toAllPlayersInZoneType(ZoneManager.java:453)
	at net.sf.l2j.gameserver.data.manager.DerbyTrackManager$Announcement.run(DerbyTrackManager.java:439)
	at net.sf.l2j.commons.concurrent.ThreadPool$TaskWrapper.run(ThreadPool.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:305)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

